rfd/00xx-ec2-tags.md

```diff
@@ -0,0 +1,35 @@
+---
+authors: Andrew Burke (andrew.burke@goteleport.com)
+state: draft
+---
+
+# RFD X - Import EC2 Instance Tags
+
+## What
+
+Teleport nodes running on EC2 instances automatically add instance tags as labels.
+
+### Related issues
+
+- [#11627](https://github.com/gravitational/teleport/issues/11627)
+
+## Why
+
+The current recommended method of [using EC2 tags as Teleport labels](https://goteleport.com/docs/setup/guides/ec2-tags/) requires
+- Creating a custom script to fetch the tags
+- Individually adding each tag as a dynamic label
+- Using the AWS API gateway, where the cost scales with the number of nodes using it
+
+As of January 2022, [instance tags are available via the instance metadata service](https://aws.amazon.com/about-aws/whats-new/2022/01/instance-tags-amazon-ec2-instance-metadata-service/). This will allow Teleport nodes to discover their own instance tags. Unlike the AWS API gateway, instance metadata requests are free and per-instance.
+
+## Details
+
+EC2 tags will be supported everywhere that dynamic labels are currently supported (i.e. SSH, Kube, Apps, and Databases).
```

Comment:
Could you clarify how this will work for K8s clusters, apps and databases? Currently, a single agent can proxy multiple databases. Will all databases proxies by an agent running on an AWS instance be automatically assigned the instance's labels?